### PR TITLE
docs: added new-test readme

### LINKED ISSUE #407: Add golden-test new-test readme doc
### TL;DR

Add golden-test new-test readme doc

### Detailed design

_No response_

### Alternatives considered

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by capri-xiyue**
fixed #407

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -140,17 +145,24 @@ Examples:
...
   same as above, but only for the specific named tests.
 
-The `<test_name>` parameter gives the test names to record or verify, if not
+For `record` and `verify` subcommand, the `<test_name>` parameter gives the test names to record or verify, if not
 specified, all tests will be run against. This flag may be repeated, like
 `--test-name=test1`, `--test-name=test2`, or `--test-name=test1,test2`.
 
-The `<location>` parameter gives the location of the template.
+For `new-test` subcommand, the `<test_name>` parameter gives the test name used to create the goldentest, like
+`--test-name=test1`, `--test-name=test2`.
```
Comment (drevell) [inline README.md:153]: For `new-test`, the test name isn't a flag, it's a positional argument. So I don't think this paragraph should say `--test-name`.

**Review by drevell — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (capri-xiyue) [reply README.md:153]: Done

File: README.md [hunk 2]
```diff
@@ -116,22 +115,28 @@ The valid values for `ABC_LOG_LEVEL` are `debug`, `info`, `notice`, `warning`,
...
-- `abc templates golden-test record [--test-name=<test_name>] <location>`
-- `abc templates golden-test verify [--test-name=<test_name>] <location>`
+- `abc templates golden-test new-test [options] --test-name=<test_name> [<location>]`
+   see `abc-templates golden-test new-test --help` for supported options.
+- `abc templates golden-test record [--test-name=<test_name>] [<location>]`
+- `abc templates golden-test verify [--test-name=<test_name>] [<location>]`
 
 Examples:
 
+- `abc templates golden-test new-test --test-name=basic examples/templates/render/hello_jupiter`
```
Comment (drevell) [inline README.md:138]: I think the same problem appears here. There is no `--test-name` flag. And also at line 131.

**Review by drevell — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (capri-xiyue) [reply README.md]: done

**Review by drevell — APPROVED** [review]

**Pull request merged**